### PR TITLE
Release 0.4.0.0

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

File: fields-json.cabal [hunk 1]
```diff
@@ -17,32 +17,31 @@ Copyright:           Scrive AB
 Stability:           Development
 Category:            Web
 Build-type:          Simple
-Cabal-version:       >=1.6
-tested-with:         GHC == 7.4.2, GHC == 7.6.3, GHC == 7.8.4, GHC == 7.10.3,
-                     GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.2
+Cabal-version:       >=1.10
+tested-with:         GHC == 8.0.2 || == 8.2.2 || == 8.4.4 || == 8.6.5
```
Comment (23Skidoo) [inline fields-json.cabal:21]: No 8.8?

**Review by 23Skidoo — APPROVED** [review]
LGTM

File: fields-json.cabal (hunk 1 above)
Comment (arybczak) [reply fields-json.cabal:21]: `json` doesn't support it yet

Comment (23Skidoo) [reply fields-json.cabal:21]: Ah ok

**Pull request merged**